main.js

```diff
@@ -0,0 +1,141 @@
+/* eslint no-param-reassign: ["error", { "props": false }] */
+const boxes = document.querySelectorAll('.box');
+
+const Player = (name, symbol) => ({ name, symbol });
+const player1 = Player('Player 1', 'X');
+const player2 = Player('Player 2', 'O');
+
+const gameBoard = (() => {
+  let board = new Array(9).fill(0);
+
+  const disable = () => {
+    boxes.forEach((box) => {
+      box.disabled = true;
+    });
+  };
+
+  const restart = () => {
+    boxes.forEach((box) => {
+      box.disabled = false;
+      box.value = '';
+    });
+    board = new Array(9).fill(0);
+  };
+
+  const turn = () => {
```

Comment:
I believe the `turn` method should be an attribute of `player`. Following the Single Responsibility principle we probably are giving our `board` too much duties. To make our `board` cleaner we can limit its methods to:

1. Initializing data (the initial array and the winning combinations array)
2. A function/method that updates each index in the array based on an input (representing the index chosen by the player).
3. A function that checks a particular moves array in comparison to the winning combinations array to find out whether there's a match -> returns true if there is and returns false if there aren't
4. A function that checks whether the array is all taken (meaning no more slots to play) -> also returns a boolean

Reply:
We removed the `turn` method from `Board` as suggested and we decide the turn in one line in the game flow function. We also removed the other methods don't belong there.